### PR TITLE
database week1 assignment

### PR DESCRIPTION
**Pull request by daxiousm**

### CODE REVIEW

File: databases/week1/hw-week1.sql [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+select title from task
+where due_date is null;
+
+ -- task 3
+select title from task
+where status_id = 3;
+
+-- task 4
+select title from task
+where status_id = 1 and 2;
```
Comment (amerdahabi) [inline databases/week1/hw-week1.sql:16]: the only data we get back here is, WHERE status_id = 1.

File: databases/week1/hw-week1.sql [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+
+-- task 5
+select * from task
+order by created desc;
+
+-- task 6
+select title from task
+order by created desc;
```
Comment (amerdahabi) [inline databases/week1/hw-week1.sql:25]: here we get all the tasks, but we need to show the single most recently created task.
you can take a look at LIMIT clause here (https://www.mysqltutorial.org/mysql-limit.aspx)

File: databases/week1/hw-week1.sql [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+ -- task 7
+ select title,due_date from task
+ where description like '%database%' 
+ or title like '%database%';
+ 
+ -- task 8
+select task.title, status.name from task
+join status on task.status_id = status_id;
```
Comment (amerdahabi) [inline databases/week1/hw-week1.sql:34]: for joining two tables we need an id from table A and reference to the column in table B, for example:
 SELECT * FROM A JOIN B ON A.id = B.a_id;
you can take a look at JOIN clause here (https://www.mysqltutorial.org/mysql-join/)

Comment (daxiousm) [reply databases/week1/hw-week1.sql:34]: yes it should be 
join status on task.status_id = status.id;

File: databases/week1/hw-week1.sql (hunk 2 above)
Comment (daxiousm) [reply databases/week1/hw-week1.sql:25]: i applied limit 3

File: databases/week1/hw-week1.sql (hunk 1 above)
Comment (daxiousm) [reply databases/week1/hw-week1.sql:16]: where status_id = 1 and  status_id = 2;

**Pull request merged**